Northstar.CustomServers/mod/scripts/vscripts/mp/_classic_mp_dropship_intro.gnut

```diff
@@ -132,22 +147,44 @@ void function EndIntroWhenFinished()
 	ClassicMP_OnIntroFinished()
 }
 
+void function SpawnPlayerOnGround( entity player )
+{
+	player.EndSignal( "OnDestroy" )
+
+	if ( IsAlive( player ) )
+		player.Die()
```

Comment:
Just wondering how the player could be alive after connecting

Reply:
sometimes they're respawned by `_base_gametype_mp`, not sure how it happens but I decided to add this check